.github/ISSUE_TEMPLATE/feature-request.md

```diff
@@ -0,0 +1,19 @@
+---
+name: Feature Request
+about: Request a new feature or request an improvement to an existing feature. 
+title: ""
+labels: New Feature
```

Comment:
Are the labels that you've used the actual labels from this project? https://github.com/paritytech/substrate-api-sidecar/labels

Reply:
@krichard410 have you looked into this?

Reply:
I think it should be fixed now! If it looks incorrect, let me know =)

Reply:
I will let @joepetrowski confirm that we are associating the correct tags with the correct Issue templates.